projects/components/src/utils/rest/rest-query-search.client.ts

```diff
@@ -8,6 +8,7 @@ import { Query, VcdApiClient, VcdHttpClient } from '@vcd/angular-client';
 import { GroupType, OrgType, UserType } from '@vcd/bindings/vcloud/api/rest/schema_v1_5';
 import { Observable } from 'rxjs';
 import { FilterBuilder } from '../filter-builder';
+import Format = Query.Format;
```

Comment:
I don't remember this import syntax, is there a reason we don't just use `Query.Format`? It's only used in one place and it avoids an import statement that is different from all other ones. Same applies to other usage